lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
I want the ordered list to follow the semantics of ruby's Array class. It has a `delete_at()` method and a `delete` method.  I see this new method being similar to that.  `Array#delete(obj)` does not delete the object, but it removes it from the array.